extensions/kusto/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -0,0 +1,276 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+import { ProviderBase } from './providerBase';
+import { SqlClusterConnection } from './connection';
+import * as utils from '../utils';
+import { TreeNode } from './treeNodes';
+import { ConnectionNode, TreeDataContext, ITreeChangeHandler } from './hdfsProvider';
+import { IFileSource } from './fileSources';
+import { AppContext } from '../appContext';
+import * as constants from '../constants';
+import * as SqlClusterLookUp from '../sqlClusterLookUp';
+import { ICommandObjectExplorerContext } from './command';
+
+export const kustoOutputChannel = vscode.window.createOutputChannel(constants.providerId);
+
+export class MssqlObjectExplorerNodeProvider extends ProviderBase implements azdata.ObjectExplorerNodeProvider, ITreeChangeHandler {
```

Comment:
probably rename `Mssql..` objects to `Kusto..`

Reply:
Changed